tests/unit/test_exceptions.py

```diff
@@ -79,6 +79,12 @@ def testCodeInvariants(self):
 
     def testInstantiation(self):
         for class_ in self._getExceptionClasses():
+            # some exceptions are becoming too complicated to instantiate
+            # like the rest of the exceptions; just do them manually
+            if class_ in [
+                    exceptions.RequestValidationFailureException,
+                    exceptions.ResponseValidationFailureException]:
+                continue
```

Comment:
@jeromekelleher the point of this test is to test that we can create Exception objects without error.  Some exception objects, however, accept arguments, so we generate dummy arguments (always the string 'arg') to pass into the object.  In the case of *ValidationFailureException, those exceptions take a json string and a class as their two arguments, so attempting to pass in 'arg' causes an exception.  That's why we need to special-case these two.  I don't see any other way around this.